python/GameOfLife/Bipeds.py

```diff
@@ -58,19 +61,3 @@ def sort_name(self):
         sorted_list.sort()
```

Comment:
Je viens de remarqué à quelle point c'est pointless ce que l'on fait ici, tant qu'à on devrait juste retourné `list(self.pop_list).sort()` On a pas besoin de faire un for loop pour creer la liste on peut juste cloner la list pop_list et imediatement la sort. C'est plus beau aussi, un peu plus compact.